### ISSUE TITLE
KeyError: 'protocols'

### ISSUE THREAD

**Issue by BatD2** [demisto/demisto-sdk#2221]
I get the errors below when running any demisto-sdk command in my environment. 
I am using custom contents repository, because github is not allowed in our environment and I must be missing some required files form the content folder. 
The commands were working until version 1.6.2 and I get the errors since 1.6.3

`$ demisto-sdk init --script -n Dummy123 -o Packs/PanoramaDevices/Scripts/
You are using demisto-sdk 1.6.9.
Please run demisto-sdk in content repository - Aborting!

Traceback (most recent call last):
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/bin/demisto-sdk", line 8, in <module>
    sys.exit(main())
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/demisto_sdk/__main__.py", line 1340, in init
    from demisto_sdk.commands.init.initiator import Initiator
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/demisto_sdk/commands/init/initiator.py", line 28, in <module>
    from demisto_sdk.commands.secrets.secrets import SecretsValidator
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/demisto_sdk/commands/secrets/secrets.py", line 15, in <module>
    from demisto_sdk.commands.common.content import Content
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/demisto_sdk/commands/common/content/__init__.py", line 6, in <module>
    from .content import *  # lgtm [py/polluting-import]
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/demisto_sdk/commands/common/content/content.py", line 12, in <module>
    from demisto_sdk.commands.common.content.objects.pack_objects.pack import Pack
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/demisto_sdk/commands/common/content/objects/pack_objects/__init__.py", line 32, in <module>
    from .pack_metadata.pack_metadata import PackMetaData
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/demisto_sdk/commands/common/content/objects/pack_objects/pack_metadata/pack_metadata.py", line 28, in <module>
    CORE_PACKS_LIST = get_core_pack_list()
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/demisto_sdk/commands/common/tools.py", line 300, in get_core_pack_list
    git_content_config=GitContentConfig(repo_name=GitContentConfig.OFFICIAL_CONTENT_REPO_NAME,
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/demisto_sdk/commands/common/git_content_config.py", line 96, in __init__
    parsed_git = GitContentConfig._get_repository_properties()
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/demisto_sdk/commands/common/git_content_config.py", line 128, in _get_repository_properties
    parsed_git = giturlparse.parse(url)
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/giturlparse/__init__.py", line 10, in parse
    return GitUrlParsed(_parse(url, check_domain))
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/giturlparse/parser.py", line 53, in parse
    matches = platform.clean_data(match.groupdict(default=""))
  File "/home/batd/PAN_WSL/Demisto_PanOS/venv/lib/python3.9/site-packages/giturlparse/platforms/base.py", line 41, in clean_data
    data["protocols"] = list(filter(lambda x: x, data["protocols"].split("+")))
KeyError: 'protocols'`

**Comment by dorschw** [comment]
Hey, thanks for opening an issue, we'll have a look and update here.

**Comment by dorschw** [comment]
The fix will be released as part of v1.7.0 next week. 
To test i before it's released, run 
```
pip install git+https://github.com/demisto/demisto-sdk.git@74ec0fad5622e06590863e848b26ce75a477cf8c
```
